v1.0/image.schema.json

```diff
@@ -9,30 +9,24 @@
   ],
   "properties": {
     "coordinateSpace": {      
-      "definition": "Coordinate space the image exists in.",
+      "_instruction": "Add the coordinate space this image exists in.",
       "_linkedTypes": [
         "https://openminds.ebrains.eu/core/CoordinateSpace"
       ]
     },
     "definedIn":{     
-      "description": "File in which the image is defined.",
+      "_instruction": "Add the file in which this image is defined in.",
```

Comment:
consistency question: I would use... is stored in."

Reply:
Yes, like in the conversation above :+1: